modules/serverless-functions-creating-on-cluster-builds.adoc

```diff
@@ -81,3 +64,27 @@ Please provide credentials for image registry used by Pipeline.
 ----
 
 . To update your function, commit and push new changes by using Git, then run the `kn func deploy --remote` command again.
+
+. Optionally, you can configure your function to be built on the cluster after every Git push by using pipelines-as-code:
+
+.. Generate the Tekton `Pipelines` and `PipelineRuns` configuration for your function:
++
+[source,terminal]
+----
+$ kn func config git set
+----
++
+Apart from generating configuration files, this command connects to the cluster and validates that the pipeline is installed. Using the token, it also creates, on behalf of the user, a webhook on the function repository. That webhook triggers the pipeline on the cluster every time changes are pushed to the repository.
++
+You need to have a valid GitHub personal access token with the repository access to use this command.
+
+.. Commit and push the generated `.tekton/pipeline.yaml` and `.tekton/pipeline-run.yaml` files:
++
+[source,terminal]
+----
+$ git add .tekton/pipeline.yaml .tekton/pipeline-run.yaml
+$ git commit -m 'Add the Pipelines and PipelineRuns configuration'
+$ git push
+----
+
+.. Once you make a change to your function, commit and push it. The function is rebuilt automatically using the created pipeline.
```

Comment:
```suggestion
.. After you make a change to your function, commit and push it. The function is rebuilt automatically by using the created pipeline.
```
IBM Style Word Usage provides this advice about _once_: "Use only to mean one time. Do not use as a conjunction to mean _after_ or _when_."